src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseViewApplicationHelper.java

```diff
@@ -0,0 +1,288 @@
+                    ADMIN_RESPONSE_ADDITIONAL_INFORMATION,
+                    reply.getAdditionalInformation());
+        }
+        return "";
+    }
+
+    private static String formatMadeByFullName(TseRespondType reply) {
+        if (!isNullOrEmpty(reply.getMadeByFullName())) { //
+            return String.format(
+                    NAME_OF_ADMIN_RESPONSE_AUTHOR,
+                    reply.getMadeByFullName());
+        }
+        return "";
+    }
+
+    private static String formatResponseDue(TseRespondType reply) {
+        if (!isNullOrEmpty(reply.getIsResponseRequired())) { //
+            return String.format(
+                    RESPONSE_DUE,
+                    reply.getIsResponseRequired());
+        }
+        return "";
+    }
+
+    private static String formatPartyOrPartiesToRespond(TseRespondType reply) {
+        if (!isNullOrEmpty(reply.getSelectPartyRespond())) { //
+            return String.format(
+                    PARTY_OR_PARTIES_TO_RESPOND,
+                    reply.getSelectPartyRespond());
+        }
+        return "";
+    }
+
+    private static String formatRespondentResponse(TseRespondType reply) {
+        return isNullOrEmpty(reply.getResponse()) ? "N/A"
+                : defaultString(reply.getResponse());
+    }
+
+    private static String formatReasonForNotSharing(TseRespondType reply) {
+        return isNullOrEmpty(reply.getCopyNoGiveDetails()) ? "N/A"
+                : defaultString(reply.getCopyNoGiveDetails());
+    }
+
+    private static DynamicValueType formatDropdownOption(GenericTseApplicationTypeItem genericTseApplicationTypeItem) {
+        GenericTseApplicationType value = genericTseApplicationTypeItem.getValue();
+        return DynamicValueType.create(value.getNumber(), String.format("%s %s", value.getNumber(), value.getType()));
+    }
+
+    private static GenericTseApplicationType getChosenApplication(CaseData caseData) {
+        return caseData.getGenericTseApplicationCollection()
+                .get(Integer.parseInt(caseData.getTseViewApplicationSelect().getValue().getCode()) - 1).getValue();
+    }
+
+    private static String getDocumentUrls(TseRespondType tseRespondType) {
+        if (tseRespondType.getSupportingMaterial() != null) {
+            return tseRespondType.getSupportingMaterial().stream()
+                    .map(doc -> createLinkForUploadedDocument(doc.getValue().getUploadedDocument())
+                            + STRING_BR)
+                    .collect(Collectors.joining(""));
+        }
+        return null;
+    }
+
+    private static String createLinkForUploadedDocument(UploadedDocumentType document) {
+        Pattern pattern = Pattern.compile("^.+?/documents/");
+        Matcher matcher = pattern.matcher(document.getDocumentBinaryUrl());
+        String documentLink = matcher.replaceFirst("");
+        String documentName = document.getDocumentFilename();
+        return String.format(DOCUMENT_LINK, documentLink, documentName);
+    }
```

Comment:
Nice refactor into its own method, though I was hoping it would be put in a more generic class so that others can use this too. There are already other methods that do this